### PR TITLE
add element_url method

### PR DESCRIPTION
**Pull request by acrogenesis**
This adds the `element_url` method :+1:

### CODE REVIEW

**Comment by rafaelfranca** [comment]
Thank you for the pull request. Could you explain why this is useful?

**Comment by acrogenesis** [comment]
On a specific case, I wanted to submit some forms directly using ajax, but I needed the url not the path. On a more general case, I without looking thru the documentation thought if there's an `element_path` sure there is an `element_url`. I believe it's the companion of `_path` & `_url` on rails routes

File: lib/active_resource/base.rb [hunk 1]
```diff
@@ -722,6 +722,10 @@ def element_path(id, prefix_options = {}, query_options = nil)
         "#{prefix(prefix_options)}#{collection_name}/#{URI.parser.escape id.to_s}#{format_extension}#{query_string(query_options)}"
       end
 
+      def element_url(id, prefix_options = {}, query_options = nil)
```
Comment (rafaelfranca) [inline lib/active_resource/base.rb]: Could you write documentation to this method?

Comment (acrogenesis) [inline lib/active_resource/base.rb]: sure, on it.

**Comment by acrogenesis** [comment]
done  6522a41

**Comment by acrogenesis** [comment]
@rafaelfranca done 6522a41

**Comment by acrogenesis** [comment]
It seems the branch still has no conflicts so it can be safely merged.

**Comment by acrogenesis** [comment]
@jeremy I think it's ready for merge

**Comment by acrogenesis** [comment]
@rafaelfranca what do you think, I believe it's mergeable.

**Comment by jeremy** [comment]
Long haul! Thank you, @acrogenesis!

**Pull request merged**